src/main/kotlin/org/rust/cargo/toolchain/RsLocalToolchain.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.toolchain
+
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.process.ProcessHandler
+import com.intellij.execution.wsl.WSLDistribution
+import com.intellij.openapi.util.SystemInfo
+import com.intellij.openapi.util.io.FileUtil
+import org.rust.cargo.runconfig.RsProcessHandler
+import org.rust.cargo.util.hasExecutable
+import org.rust.cargo.util.pathToExecutable
+import org.rust.stdext.isExecutable
+import java.io.File
+import java.nio.file.Path
+
+object RsLocalToolchainProvider : RsToolchainProvider {
```

Comment:
Note, `object` makes no sense in case of extension because the platform creates instance via reflection and it differs from generated one by `object` mechanics